test/unit/connectors/ExportsConnectorSpec.scala

```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
 
     "when making a successful request" should {
 
-      "use stub service url" in {
+      "ensure URL is retrieved from config" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         verify(mockWsPost).POSTString(ameq(serviceConfig.url), anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), any[HeaderCarrier](), any[ExecutionContext])
       }
 
-      "pass the xml in the body" in {
+      "ensure xml payload is included in the request body" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         verify(mockWsPost).POSTString(anyString, ameq(xml.toString()), any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), any[HeaderCarrier](), any[ExecutionContext])
       }
 
-      "set the content type header" in {
+      "ensure the content type header in passed through in the request" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.CONTENT_TYPE -> (MimeTypes.XML + "; charset=UTF-8"))
       }
 
-      "set the accept header" in {
+      "ensure the accept header in passed through in the request" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.ACCEPT -> MimeTypes.XML)
       }
 
-      "set the date header" in {
+      "ensure the date header in passed through in the request" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         val headersCaptor: ArgumentCaptor[HeaderCarrier] = ArgumentCaptor.forClass(classOf[HeaderCarrier])
         verify(mockWsPost).POSTString(anyString, anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), headersCaptor.capture(), any[ExecutionContext])
         headersCaptor.getValue.extraHeaders should contain(HeaderNames.DATE -> httpFormattedDate)
       }
 
-      "set the X-Forwarded-Host header" in {
+      "ensure the X-FORWARDED_HOST header in passed through in the request" in {
```

Comment:
as above "ensure the X-FORWARDED_HOST header is...."

Reply:
Fixed